homeassistant/helpers/integration_platform.py

```diff
@@ -194,6 +194,31 @@ async def async_process_integration_platforms(
     if not top_level_components:
         return
 
+    # We create a task here for two reasons:
+    #
+    # 1. We want the integration that provides the integration platform to
+    #    not be delayed by waiting on each individual platform to be processed
+    #    since the integration platforms may have to schedule I/O or executor
+    #    jobs.
+    #
+    # 2. We want the behavior to be the same as if the integration that has
+    #    the integration platform is loaded after the platform is processed.
+    #
+    hass.async_create_task(
```

Comment:
We use a tracked task because we still want to wait for it to load before the start event

Reply:
a8af6e02935118c4486ff70d98d3f0026d1062f6